xcodeproj/internal/xcodeproj_rule.bzl

```diff
@@ -945,17 +949,18 @@ targets.
 
 def _process_xcode_generated_paths(
         *,
-        ctx,
+        actions,
         build_mode,
+        bwx_unfocused_dependencies,
         focused_targets,
-        bwx_unfocused_dependencies):
+        name):
     xcode_generated_paths = {}
-    xcode_generated_paths_file = ctx.actions.declare_file(
-        "{}-xcode_generated_paths.json".format(ctx.attr.name),
+    xcode_generated_paths_file = actions.declare_file(
```

Comment:
Is this `actions` as param vs. declaring new actions via `ctx.actions` a common pattern amongst the rulesets? Doesn't change functionality, it seems, but it's odd on the surface for me.

Reply:
https://docs.google.com/document/d/1uuX1Gz6Kvivp-oOqGja342VrVO534PLF0m_zFtFIBcc/edit#bookmark=id.wqzr2iuioeu8